app/views/errors/UnauthorisedAgentAccessView.scala.html

```diff
@@ -35,11 +37,24 @@
     }
 
     @p {
-       @messages("unauthorised.agent.para2.1")
-       <a href="@appConfig.signOutUrl">@messages("unauthorised.agent.signInAgain")</a>
-       @messages("unauthorised.agent.para2.2")
+        <a href="@appConfig.signOutUrl">@messages("unauthorised.agent.signInAgain.1")</a>
+        @messages("unauthorised.agent.signInAgain.2")
     }
 
+  @bullets(Seq(
+      messages("unauthorised.agent.bullet.1"),
+      messages("unauthorised.agent.bullet.2")
+  ).map(Html(_)))
+
+    @govukDetails(
+        Details(
+            summary = HtmlContent(messages("unauthorised.agent.details.linkText")),
+            content = HtmlContent(
+                messages("unauthorised.agent.details.1") + "<br/>" + "<br/>" + messages("unauthorised.agent.details.2")
```

Comment:
in the prototype these are 2 paragraphs, you can probably do it like this:
```suggestion
                <p>{messages("unauthorised.agent.details.1")}</p> +
                <p>{messages("unauthorised.agent.details.2")}</p>.toString()
```